src/test/java/by/innowise/monolithwebcalculator/web/controller/CalculatorControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package by.innowise.monolithwebcalculator.web.controller;
+
+import by.innowise.monolithwebcalculator.domain.mapper.OperationMapper;
+import by.innowise.monolithwebcalculator.domain.operation.Operation;
+import by.innowise.monolithwebcalculator.domain.operation.OperationType;
+import by.innowise.monolithwebcalculator.service.CalculatorService;
+import by.innowise.monolithwebcalculator.web.dto.OperationDto;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import javax.servlet.ServletContext;
+
+import static org.assertj.core.internal.bytebuddy.matcher.ElementMatchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@WebMvcTest(CalculatorController.class)
+@ExtendWith(MockitoExtension.class)
```

Comment:
I doubt you need `MockitoExtension` here. `@MockBean` should be a part of spring boot